jest.config.js

```diff
@@ -13,7 +13,10 @@ const customJestConfig = {
   // if using TypeScript with a baseUrl set to the root directory then you need the below for alias' to work
   moduleDirectories: ['node_modules', '<rootDir>/'],
   testEnvironment: 'jest-environment-jsdom',
-  testPathIgnorePatterns: ['/node_modules/', '/__tests__/helpers/']
+  testPathIgnorePatterns: ['/node_modules/', '/__tests__/helpers/'],
+  moduleNameMapper: {
+    '^uuid$': require.resolve('uuid')
```

Comment:
ofc. It's because it's a dependency of fqm-execution, which is installed in this PR. Ran into some issues getting the dependency to resolve and came to this solution with Hoss.